ScriptingEngineTester/src/main/resources/zencode_tests/stdlib/exception_subtype_0.zc

```diff
@@ -0,0 +1,13 @@
+#dependency: stdlib
+#output: Hello world
+
+import stdlib.Exception;
+
+public virtual class MyException : Exception {
```

Comment:
Test fails since JavaPrepareDefinitionVisitor tries to prepare Exception a 2nd time in the `script_tests` module while preparing `MyException`